README.md

```diff
@@ -356,12 +338,20 @@ MemoryLocation can be used if you don't want router to touch the address bar at
 e.g.
 
 ```js
-var cherrytree = require('cherrytree')
-var MemoryLocation = require('cherrytree/lib/locations/memory')
-var router = cherrytree()
-routerlisten(new MemoryLocation())
+var router = cherrytree({
+  location: 'memory'
```

Comment:
Ah yes it lines up with `document.location` , right. Yes perhaps the latter rename is fine.